packages/perf/package.json

```diff
@@ -29,9 +30,6 @@
   "devDependencies": {
     "serve": "^14.2.0"
   },
-  "resolutions": {
-    "serve/boxen/type-fest": "^2.13.0"
-  },
```

Comment:
@kumavis Do you have any memory of the reason behind type-fest being resolved to 2.13 in perf and to 0.21 pretty much everywhere else?

Reply:
unless something is using `tsc` to build or check types when running things in `perf`, I'm unsure how this could be a problem.  `type-fest` contains no code and does nothing